### PR TITLE
Update Guweb version to 1.3.0

### PR DESCRIPTION
**Pull request by Varkaria**
Update Lists
- No more guweb API (just use gulag API only)
- Profile Customisation (Banner & Background Profile)
image
- Change a bit UI of Profile & Leaderboard Page
- Redesign Settings page
- Bug fixed
- More cleaning code

### CODE REVIEW

**Comment by Varkaria** [comment]
@Gusb3ll @JKBGL @cmyui Go check and approve this please if it's working well same as main branch

File: blueprints/frontend.py [hunk 1]
```diff
@@ -2,49 +2,60 @@
...
 from quart import request
 from quart import session
+from quart import send_file
 
 from constants import regexes
 from objects import glob
 from objects import utils
 from objects.privileges import Privileges
-from objects.utils import flash
+from objects.utils import flash, flash_with_customizations
```
Comment (cmyui) [inline blueprints/frontend.py:26]: The way I originally ordered the imports follows [this](https://www.youtube.com/watch?v=Sjor8PZXnaw&list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY), basically:
```py
# first; stdlib imports
import stdlib_module
from stdlib_module import A
from stdlib_module import b

# second; third party imports
import third_party_module
from third_party_module import C
from third_party_module import d

# third; first party imports (imports from our project)
import first_party_module
from first_party_module import E
from first_party_module import f
```
I split up each import into it's own line to [pervent merge conflicts](https://github.com/asottile/reorder_python_imports#why-this-style), and order them alphabetically just for style.

I do this for all my projects now.

**Comment by cmyui** [comment]
Changes look nice though :)

**Review by Gusb3ll — APPROVED** [review]
Everything looks fine for me 🍙

**Pull request merged**